spec/minimal_repo.rb

```diff
@@ -5,13 +5,7 @@ module MinimalRepo
   class << self
     DIRECTORIES = [
       '.git',
-      'config',
-      'hosts',
-      'genders',
-      'dhcp',
       'files',
```

Comment:
We need it for those as we only want to create an empty directory in those cases. I guess we could specify a file we don't actually want in those directories and then have `create_files_at` implicitly create the directory as well, but it seems that would make things slightly confusing in order to remove a couple of lines of code.